kratos/includes/stream_serializer.h

```diff
@@ -0,0 +1,67 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+//
+
+#if !defined(KRATOS_STREAM_SERIALIZER_H_INCLUDED )
+#define  KRATOS_STREAM_SERIALIZER_H_INCLUDED
+
+// System includes
+#include <string>
+#include <cstring>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/serializer.h"
+
+//This is a Helper object to provide a simpler interface for serialization to a string instead of to a file
```

Comment:
//This class provides a simpler interface for serialization to a string instead of to a file
// Note that you may not override any load or save method of the Serializer. They are not virtual.